app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.scss

```diff
@@ -2,12 +2,12 @@
 
 .document-capture-selfie-feedback {
   color: color('white');
-  background: color('black');
+  background: rgba(0, 0, 0, .5);
   left: 50%;
   top: 10%;
   position: fixed;
   transform: translateX(-50%);
-  border-radius: 5px;
-  padding: 10px;
+  padding: 0 5px 0 5px;
+  opacity: 100%;
```

Comment:
Does this do anything?

```suggestion
```

Reply:
I think @charleyf made this change because he wanted the opacity to match the document capture hint text?  Slack convo [here](https://gsa-tts.slack.com/archives/C056RD1NEHW/p1711035090031239?thread_ts=1711031776.441309&cid=C056RD1NEHW).

Reply:
Elements are 100% opacity by default. I don't see any place where it'd be anything other than 100%?

I think we want to change the _background_ value to avoid using `rgba` if we want it to not appear semi-transparent.

Reply:
This `opacity: 100%;` is a mistake. I left it in by accident because I was experimenting with setting CSS on the container and the `<p>`. [Removed it](https://github.com/18F/identity-idp/pull/10274/commits/b3550f7e54b8cbd006df1054eeb4671dcf597b07).

- The goal is (currently as I understand it) to have the background be semi-transparent to match the document capture hint text.
- That's achieved with `background: rgba(0, 0, 0, .5);`
- That is not equivalent to `background: color(black); opacity: 50%;` because that makes the white text also 50% transparent, which makes it very hard to read (and less accessible).